rules/windows/process_creation/win_susp_use_of_sqlps_bin.yml

```diff
@@ -0,0 +1,28 @@
+title: Detection of PowerShell Execution via SQL 
+id: 0152550d-3a26-4efd-9f0e-54a0b28ae2f3  
+status: experimental
+description: PowerShell execution through builtin SQL Server "SQLPS.exe" binary. Microsoft PS logging like
+  ScriptBlock logging function of PowerShell is not an option here, PS session caused by the binary won't be recorded/logged.
+references:
+  - https://twitter.com/pabraeken/status/993298228840992768
+  - https://docs.microsoft.com/en-us/sql/tools/sqlps-utility?view=sql-server-ver15 
+  - https://lolbas-project.github.io/lolbas/OtherMSBinaries/Sqlps/
+tags:
```

Comment:
I also believe that this utility will be used to bypass security systems, since modules and script blocks are not logged with this utility. This utility can be considered the developer's trusted utility.

Therefore, I propose to additionally add `defense_evasion` tactic and `Trusted Developer Utilities Proxy Execution` technique (T1127).